doc/getting-started.rst

```diff
@@ -166,12 +176,11 @@ Now you can build and re-run your code to see the new output:
         /    /  /    /   \    \
        /____/  /____/     \____\
 
-Run a single-file Haskell script
---------------------------------
+Running a single-file Haskell script
+------------------------------------
 
-Cabal also enables us to run single-file Haskell scripts
-without creating a project directory or ``.cabal`` file.
-The cabal directives are placed in the file within a comment.
+Cabal also supports to run a single self-contained Haskell script like 
+the following file named ``myscript``:
```

Comment:
I was kind of expecting the haskell-say example to be used here. It would feel more like a natural continuation of what has been written so far.

Reply:
Good point. I made the examples the same.